.gitbook.yml

```
@@ -40,3 +40,5 @@ redirects:
   help/ops-home: ./help/operations.md
   development: ./help/operations/architecture.md
   development/writing-parsers: ./data-onboarding/custom-log-types/README.md
+  #Moving Alert Runbooks to Triaging Alerts
+  alert-runbooks*: ./writing-detections/triaging-alerts/alert-runbooks/*
```

Would that be what we're after?
```suggestion
  alert-runbooks/*: ./writing-detections/triaging-alerts/alert-runbooks/*
```

---

@3nvi thanks for checking this out! I actually just created this as an example. Talking about this with Mo in #web-help and following up with GitBook support because we don't think it's possible, but you're right about that slash being needed if it is.